.github/workflows/release.yml

```diff
@@ -19,7 +19,9 @@ env:
   # renovate: datasource=github-releases depName=kubernetes-sigs/controller-tools
   CONTROLLER_TOOLS_VERSION: "v0.15.0"
   SCHEDULER_COMPATIBLE_K8S_VERSION: "v0.24.3"
-  PUBLISHABLE_ITEMS: '[".","lifecycle-operator","metrics-operator","keptn-cert-manager","runtimes/deno-runtime","runtimes/python-runtime","scheduler"]'
+  PUBLISHABLE_ITEMS: >
+    '[".","lifecycle-operator","metrics-operator","keptn-cert-manager",
```

Comment:
since we use `>`, I don't think we need `'` anymore
I suggest sorting one item per line :)